wfe2/verify.go

```diff
@@ -369,6 +369,18 @@ func (wfe *WebFrontEndImpl) parseJWS(body []byte) (*bJSONWebSignature, error) {
 	bodyStr := string(body)
 	parsedJWS, err := jose.ParseSigned(bodyStr, getSupportedAlgs())
 	if err != nil {
+		// Special case needed to send right type of error when parser errors because of
+		// wrong signature algorithm.
```

Comment:
Ideally this comment is unnecessary, I think the code below explains itself quite well.

```suggestion
```